x-pack/plugins/security_solution/common/detection_engine/constants.ts

```diff
@@ -27,3 +27,8 @@ export enum RULE_PREVIEW_FROM {
 }
 
 export const PREBUILT_RULES_PACKAGE_NAME = 'security_detection_engine';
+
+/**
+ * Rule signature id (`rule.rule_id`) of the prebuilt "Endpoint Security" rule.
+ */
+export const ELASTIC_SECURITY_RULE_ID = '9a1a2dae-0b5f-4c3d-8305-a268d404c306';
```

Comment:
This is a refactor. I moved this from the Detection's FTR tests to here so that it can be re-used by our cypress suite